package.json

```diff
@@ -8,5 +8,12 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
-}
\ No newline at end of file
+  "license": "ISC",
```

Comment:
Can we revert all changes to both this file and delete the package-lock.json at the root of the repo please